local-tests/setup/shiva-client.ts

```diff
@@ -281,7 +289,11 @@ export class ShivaClient {
         }
       );
 
-      const createTestnet = await createTestnetResp.json();
+      const createTestnet =
+        (await createTestnetResp.json()) as TestNetResponse<void>;
+      if (createTestnetResp.status != 200) {
```

Comment:
I think this should go before trying to parse the result as json, so if we get an e.g. error `500` back with a string as its body, we don't throw a failure to parse the body as JSON before we check the response status?